docs/config.md

```diff
@@ -215,13 +218,13 @@ analysis_models: <comma-delimited-string-list>
 [ filename_server_only: <string> | default: metrics-server-only.csv ]
 
 # Specifies columns keys for model inference metrics table
-[ inference_output_fields: <comma-delimited-string-list> | default: See Config Defaults section]
+[ inference_output_fields: <comma-delimited-string-list> | default: See [Config Defaults](#config-defaults) section]
```

Comment:
I think it would be better if we don't exposes these parameters inside the documentation. Having `--metrics` flag, appropriately set these parameters is a better option.

Reply:
The `--metrics` flag is replaced with `--collect-cpu-metrics` flag in flavor for simplicity, so less parameters are exposed.

Reply:
`--collect-cpu-metrics` sounds good as well.